azure-pipelines.yml

```diff
@@ -31,10 +31,15 @@ jobs:
     displayName: 'Install dependencies'
 
   - script: |
-      pip install pytest pytest-azurepipelines
-      pytest --verbose
+      pip install pytest pytest-azurepipelines pytest-cov
```

Comment:
would it make sense to add these and all of the linters to `requirements.txt`? Or should we rather maintain a separate CI requirements file?

Reply:
Let's do that in a separate PR.